redbot/core/commands/converters.py

```diff
@@ -0,0 +1,80 @@
+import re
+from datetime import timedelta
+from typing import TYPE_CHECKING
+
+from redbot.core.commands import Converter, BadArgument
+
+if TYPE_CHECKING:
+    from .commands import Context
+
+__all__ = ["TimedeltaConverter"]
```

Comment:
Not sure why dunder all is needed here. TimedeltaConverter is literally the only thing here. Typically you use all to "hide" some of the helper functions that build up the stuff you want developers to use.

Reply:
I followed `converters.py` as an example for this PR, and tried to match with their style, but if it's not needed I'll remove it 👍